file_zbc.c

```diff
@@ -951,13 +951,14 @@ static int zbc_evpd_inquiry(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 
 	case 0x00:
 		/* Supported VPD pages */
-		data[3] = 5;
-		data[4] = 0x83;
-		data[5] = 0xb0;
-		data[6] = 0xb1;
-		data[7] = 0xb6;
-
-		tcmu_memcpy_into_iovec(iovec, iov_cnt, data, 8);
+		data[3] = 6;
```

Comment:
Shouldn't this be 5 ?

Reply:
The PAGE LENGTH  should be `n-3` and `n == 8`.

Reply:
I agree with this, and that explains the `4c` and `00` trailing bytes in both of my prior posted examples. I just incremented all the numbers by 1, so this must be an artifact of the original implementation.

Reply:
Tested and pushed.

```
root@mymachine:~# sg_vpd -p 0x0 /dev/sdf -rr | xxd
00000000: 1400 0005 0083 b0b1 b6                   .........
```

Reply:
Yeah, the first one, the data[4], should always be `0x00`.